iped-app/resources/localization/iped-desktop-messages.properties

```diff
@@ -394,4 +394,44 @@ TimeLineGraph.uncheckEventItems=Uncheck corresponding event type items
 IntervalDefinitionDialog.Title=Time line chart defined ranges
 IntervalDefinitionDialog.dateFormat=MM/dd/yyyy
 IntervalDefinitionDialog.hourFormat=HH:mm:ss
-IntervalDefinitionDialog.Exit=Exit
\ No newline at end of file
+IntervalDefinitionDialog.Exit=Exit
+App.appliedFilters=Applied Filters
+iped.app.ui.CategoryTreeListener.filtererName=Category panel filterer
+iped.app.ui.FilterManager.filtererName=Predefined filters
+iped.app.timelinegraph.IpedChartsPanel.filtererName=Timeline chart panel
+iped.app.ui.App.DuplicatesFilterer.filtererName=Dupicates filterer
+iped.app.ui.App.SimilarFacesSearchFilter.filtererName=Similar faces filterer
+iped.app.ui.App.SimilarImagesQueryFilterer.filtererName=Similar images filterer
+iped.app.ui.App.SearchFilterer.filtererName=Search input
+iped.app.ui.TableHeaderFilterManager.filtererName=Results table header filterer
+iped.app.ui.TreeListener.filtererName=Evidence panel
+iped.app.ui.ComboFilterer.filtererName=Predefined filters combobox
+iped.app.ui.ComboFilterer.tooltip=Select a predefined filter
+iped.app.ui.BookmarksTreeListener.filtererName=Bookmarks panel
+iped.app.ui.filterdecisiontree.CombinedFilterer.filtererName=Combined filters filterer
+iped.app.ui.filterdecisiontree.CombinedFilterer.tooltip=Apply combined filter to resultset.
+iped.app.graph.FilterSelectedEdges.filtererName=Selected graph edges filterer
+iped.app.ui.MetadataPanel.filtererName=Metadata panel
+iped.app.ui.TimelineListener.filtererName=Timeline view button
+iped.app.ui.App.SimilarDocumentFilterer.filtererName=Similar documents filterer
```

Comment:
Hi @patrickdalla, sorry to disturb you in your vacation. But are these localization keys needed? They are missing in other localization files and there is no direct reference to them in the code, but seems some dynamic loading is being done... Should I delete them or replicate all or part of them to other localization files?

Reply:
I reviewed quickly (in my phone) the code and the filterers names are hard coded. It seems i began to plan localization but I have not finished.